client/gutenberg/editor/controller.js

```diff
@@ -262,13 +262,15 @@ export const post = ( context, next ) => {
 export const siteEditor = ( context, next ) => {
 	const state = context.store.getState();
 	const siteId = getSelectedSiteId( state );
+	const unlockStyles = 'unlock-styles' in context.query;
 
 	context.primary = (
 		<CalypsoifyIframe
 			// This key is added as a precaution due to it's oberserved necessity in the above post editor.
 			// It will force the component to remount completely when the Id changes.
 			key={ siteId }
 			editorType="site"
+			unlockStyles={ unlockStyles }
```

Comment:
This is a question, not a suggestion. Could we use just `unlockStyles` in this case instead of `unlockStyles={unlockStyles}`?

Reply:
Nope, because that would always define a `unlockStyles` property regardless of what's in `context.query`.